core/chains/evm/txmgr/models.go

```diff
@@ -31,6 +33,8 @@ type EthTxMeta struct {
 	// Used for the VRFv2 - the subscription ID of the
 	// requester of the VRF.
 	SubID uint64 `json:"SubId"`
```

Comment:
Not related to this change, just curious, this field seems specific to vrf, is it included in all logs with some default value?

Reply:
Was thinking about that as well. Updated that logic.